phys/module_radiation_driver.F

```diff
@@ -1801,7 +1822,20 @@ SUBROUTINE radiation_driver (                                  &
              ENDIF
 
         CASE (RRTMG_LWSCHEME)
+
              CALL wrf_debug (100, 'CALL rrtmg_lw')
+             IF ( mp_physics .EQ. nuwrf4icescheme ) THEN
+                DO j=jts,jte
+                DO k=kts,kte
+                DO i=its,ite
+                   re_cloud(i,k,j) = re_cloud_gsfc(i,k,j) * 1.E-6
```

Comment:
Just checking ... Should these be divides instead of multiplies?

Reply:
Yes, this is correct.

Reply:
Thanks for checking